.github/pull_request_template.md

```diff
@@ -5,7 +5,8 @@
 *Please write why your merge request is needed*
 
 # Other notes
-*Please write if you have anything*
+Closes #*issue_number*
+*Please write if you have anything to add*
```

Comment:
You should add an extra blank line between those two lines, otherwise Markdown will render them without a line break in between.

Reply:
Done